azure-pipelines.yml

```diff
@@ -556,27 +561,21 @@ stages:
             - task: DownloadPipelineArtifact@2
               inputs:
                 buildType: 'current'
-                itemPattern: |
-                  *.zip
-                  version.txt
-                targetPath: '$(Pipeline.Workspace)\\scanner-packages'
```

Comment:
Changed the path for the artifact to be restored to the original location where it was stored from. So it's always on the same location for local run and CI run